utils/dates.js

```diff
@@ -16,16 +16,29 @@ module.exports = {
 
 	/**
 	 * UTC timezone offset in minutes
-	 * @atDate - Effective date for the offset
+	 * @atDate - Effective date for the offset (MM/DD/YYYY, YYYY-MM-DD, moment, or default to today if null)
 	 * @return {int} offset
 	 */
 	timezoneOffset: function(atDate) {
-		var dt = atDate ? atDate : moment().format("YYYY-MM-DD");
+		var dt = "";
+		if (atDate) {
+			if (typeof atDate == "string") {
+				if (atDate.indexOf("/") == -1)  {
+					dt = atDate;   // String in YYYY-MM-DD already
+				} else {
+					dt = moment(atDate, "MM/DD/YYYY").format("YYYY-MM-DD");   // String in MM/DD/YYYY
+				}
+			} else {
+				dt = atDate.format("YYYY-MM-DD");    // Moment object
```

Comment:
Looks good to me.  But just trying to understand this part,  are we expecting a moment object to be passed in as an argument for the atData parameter in a special use case?